sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveQuerySuite.scala

```diff
@@ -467,6 +467,7 @@ class HiveQuerySuite extends HiveComparisonTest with BeforeAndAfter {
 
   test("sampling") {
     sql("SELECT * FROM src TABLESAMPLE(0.1 PERCENT) s")
+    sql("SELECT * FROM src TABLESAMPLE(100 PERCENT) s")
```

Comment:
I'm going to go ahead and merge this since it changes semantics and we are close to the release where we remove the alpha tag, but it would be great if you could add a test that actually checks to make sure sampling is happening and we are getting something close to the expected number of results.